python/custreamz/custreamz/tests/test_dataframes.py

```diff
@@ -1,9 +1,6 @@
 """
-Tests for cudf DataFrames:
-All tests have been cloned from the test_dataframes module in the streamz
-repository. Some of these tests pass with cudf, and others are marked with
-xfail, where a pandas-like method is not yet implemented in cudf. But these
-tests should pass as and when cudf rolls out more pandas-like methods.
+Tests for Streamz Dataframes (SDFs) built on top of cuDF DataFrames:
+*** Borrowed from streamz.dataframe.tests ***
```

Comment:
If we're copying the tests from Streamz we need to include the license from Streamz: https://github.com/python-streamz/streamz/blob/master/LICENSE.txt

Reply:
Does modifying the tests to use cudf instead of Pandas also mean copying? If so, I can add the license. Do I need to add a link to the license before the tests, or add a separate file and copy the license in it?

Reply:
> Does modifying the tests to use cudf instead of Pandas also mean copying? If so, I can add the license. Do I need to add a link to the license before the tests, or add a separate file and copy the license in it?

Yes. You need to copy the license file somewhere into the cuDF repo, but I'm not quite sure as to where. 

@fondaing can you help to advise here? Does the SWIPAT we have give guidance as to where we should put any thirdparty licenses of code that we distribute from other libraries?

Reply:
The process says you can just put IN the repo - it does not specify a location.  We can probably put where we like.  I have seen them named thridpary/license  in the past.  Also remember we have to include the copyright and attributions in the header - here is a example 
https://gist.github.com/Rypac/7349957f95f0713143fda2f3d3bc10c8.

Reply:
Done

Reply:
Please add NVIDIA copyright at the top of this file and indicate in the docstring to look in `thirdparty/LICENSE` for the `streamz` license.